src/Backend/Modules/Mailmotor/Installer/Installer.php

```diff
@@ -61,9 +61,12 @@ private function configureFrontendPages(): void
     {
         // loop languages
         foreach ($this->getLanguages() as $language) {
-            $pageId = $this->insertPage(
-                ['title' => 'Newsletters', 'language' => $language]
-            );
+            $pageId = $this->getPageWithMailmotorBlock($language);
+            if ($pageId === null) {
+                $pageId = $this->insertPage(
+                    ['title' => 'Newsletters', 'language' => $language]
```

Comment:
Why split all parameters to new lines, but leave the full array on a single line?